utils/cliutils/utils.go

```diff
@@ -415,6 +415,10 @@ func GetSpec(c *cli.Context, isDownload, overrideFieldsIfSet bool) (specFiles *s
 	return
 }
 
+func CreateSpecFromBuildNameAndNumber(buildName string, buildNumber string) (specFiles *speccore.SpecFiles, err error) {
```

Comment:
This wrapper function seema redundant. The `speccore.CreateSpecFromBuildNameAndNumber` can be called directly.